Assets/Scripts/Enemy.cs

```diff
@@ -45,7 +45,7 @@ void FixedUpdate ()
 		}
 
 		// Set the enemy's velocity to moveSpeed in the x direction.
-		rigidbody2D.velocity = new Vector2(transform.localScale.x * moveSpeed, rigidbody2D.velocity.y);	
+		GetComponent<Rigidbody2D>().velocity = new Vector2(transform.localScale.x * moveSpeed, GetComponent<Rigidbody2D>().velocity.y);	
```

Comment:
Looking at this PR I spotted the use of GetComponent inside update and fixed update loops. How do you think this could be made more efficient?

Reply:
Instead of calling GetComponent<Rigidbody2D>() every time it loop, have a reference variable so you only need to call GetComponent<Rigidbody2D>()  once and use that reference variable to update the velocity.